src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -13,9 +13,12 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
   // Item
   --pf-c-toolbar__item--Display: block;
   --pf-c-toolbar__item--MinWidth--base: auto;
+  --pf-c-toolbar__item--AlignSelf: baseline;
```

Comment:
should this be `auto`? otherwise, setting `align-items` on the group doesn't seem like it will ever work, since `align-self` on the items will always override that.